gm4_shiny_items/data/gm4_shiny_items/functions/blacklist_items.mcfunction

```diff
@@ -0,0 +1,127 @@
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:light_gray_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:cyan_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:purple_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:blue_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:brown_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:green_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:red_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:black_shulker_box"}}] run tag @s add gm4_shiny_blacklist
+# Beds
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:white_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:orange_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:magenta_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:light_blue_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:yellow_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:lime_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:pink_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:gray_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:light_gray_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:cyan_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:purple_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:blue_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:brown_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:green_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:red_bed"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:black_bed"}}] run tag @s add gm4_shiny_blacklist
```

Comment:
I don't know if there is a better way of doing this, but this will be a nightmare to maintain

Reply:
Could copy it to I think it was an armour stand and then use an item tag check? Don't know how performant it would be

Reply:
I have no idea how to do that, also I don't quite understand what you mean by "nightmare to maintain". It is already done and doesn't need to be changed.

Reply:
Bloo is talking about for future versions. I.E. whenever new items get added to the game